librepo/downloader.c

```diff
@@ -991,9 +991,20 @@ has_librepo_xattr(int fd)
 /** Remove Librepo extended attribute
  */
 static void
-remove_librepo_xattr(int fd)
+remove_librepo_xattr(LrDownloadTarget * target)
 {
+    int fd = target->fd;
+    if (fd != -1) {
+        fremovexattr(fd, XATTR_LIBREPO);
+        return;
+    }
+    // File descriptor was not provided to targed therefore open file is required
```

Comment:
The comment could use some English improvements, but I don't find it particularly helpful - what about something like:
> If file descriptor wasn't set, file name was, and we need to open it